packages/apollo-federation/src/service/printFederatedSchema.ts

```diff
@@ -347,7 +347,12 @@ function printDescription(
   }
 
   const lines = descriptionLines(def.description, 120 - indentation.length);
-  if (lines.length === 1) {
+
+  // If a description uses the double quote character, it absolutely must use
+  // a multi-line comment.
+  const requireMultiline = lines.some(line => line.includes('"'));
+
+  if (lines.length === 1 && !requireMultiline) {
```

Comment:
Escaped quotes (i.e. `\"`) are valid in non-`block`'d `def.description.value`s in a schema's AST, so the comment here is not quite correct.  Further, I believe by using this implementation we'd be applying/requiring this transformation unnecessarily.

Perhaps we might consider downstreaming some of the changes in `graphql`'s `printSchema` which seems to have [a more current version of the `printDescription` method](https://github.com/graphql/graphql-js/blame/af480089b5fcaa961adb356ef1dca13cf475e75f/src/utilities/printSchema.js#L324-L345) which I believe we _cargo cult_-ed at one point.

Generally, except for where we leverage specific overrides, I'd have a preference for a more exact copy of that implementation.